src/main/java/org/opensrp/web/config/SwaggerConfig.java

```diff
@@ -54,8 +65,14 @@ public ApiInfo getApiInfo() {
 
     @Override
     public void addResourceHandlers(ResourceHandlerRegistry registry) {
-        registry.addResourceHandler("swagger-ui.html").addResourceLocations("classpath:/META-INF/resources/");
-        registry.addResourceHandler("/webjars/**").addResourceLocations("classpath:/META-INF/resources/webjars/");
+        registry.addResourceHandler("/swagger-ui/**")
+                .addResourceLocations("classpath:/META-INF/resources/webjars/springfox-swagger-ui/");
```

Comment:
Why were the `/webjars` resourcehandler and resourceLocation removed?

Reply:
Apart from swagger-ui is there any feature that needs it.

Reply:
Not that I'm aware of

Reply:
Seems it was specifically added for swagger-ui.[commit](https://github.com/opensrp/opensrp-server-web/commit/70ecca7d277047add3753ebf1a7d34857ec182b1)